dojo/models.py

```diff
@@ -3294,8 +3294,7 @@ def inherit_tags(self, potentially_existing_tags):
 
     @property
     def violates_sla(self):
-        days_remaining = self.sla_days_remaining()
-        return days_remaining < 0 if days_remaining else False
+        return (self.sla_expiration_date and self.sla_expiration_date > timezone.now())
```

Comment:
Looks like this should be reversed:
```suggestion
        return (self.sla_expiration_date and self.sla_expiration_date < timezone.now())
```